app/src/pages/playground/ModelConfigButton.tsx

```diff
@@ -204,12 +223,18 @@ function ModelConfigDialog(props: ModelConfigDialogProps) {
       size="M"
       extra={
         <TooltipTrigger delay={0} offset={5}>
-          <Button size={"compact"} variant="default" onClick={onSaveConfig}>
-            Save Config
+          <Button
+            size={"compact"}
+            variant="default"
+            onClick={onSaveConfig}
+            icon={<Icon svg={<Icons.SaveOutline />} />}
+          >
+            Save as Default
           </Button>
           <Tooltip>
-            Remember configuration for{" "}
-            {ModelProviders[instance.model.provider] ?? "this provider"}.
+            Saves the current configuration as the default for{" "}
+            {ModelProviders[instance.model.provider] ?? "this provider"} for
```

Comment:
The "for later use" kinda makes this message a tongue twister.

Two back to back "for"s feels redundant

Reply:
yeah i agree maybe will just leave it in th etoast but remove here